### ISSUE TITLE
031

### ISSUE THREAD

**Issue by just-nana** [thendo991/CleanCity#31]
No alt for an image.

Expected: All images should have "alts" for screen readers.
Actual: This image does not have an "alt".

Main index.html file

<img width="1531" height="304" alt="Image" src="https://github.com/user-attachments/assets/555566a4-8308-471c-b6b4-19037986222f" />
